Formula/t/tdb.rb

```diff
@@ -0,0 +1,27 @@
+class Tdb < Formula
+  desc "Trivial DataBase, by the Samba project"
+  homepage "https://tdb.samba.org/"
+  url "https://www.samba.org/ftp/tdb/tdb-1.4.10.tar.gz"
+  sha256 "02338e33c16c21c9e29571cef523e76b2b708636254f6f30c6cf195d48c62daf"
+  license "GPL-3.0-or-later"
+
+  uses_from_macos "python" => :build
+  on_linux do
+    depends_on "python@3.12" => :build
+  end
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
```

Comment:
would this work?
```suggestion
    system "./configure", *std_configure_args
```

Reply:
No. Tdb doesn't use autotools but waf as a build chain. It doesn't recognise the standard arguments